src/cli.rs

```diff
@@ -93,6 +93,11 @@ pub struct WalletConfig {
     /// Signet: 50001, regtest: 60401
     #[arg(long = "wallet-electrum-port")]
     pub electrum_port: Option<u16>,
+
+    /// If no existing wallet is found, automatically initialize a new one.
+    /// Creates an unencrypted, plaintext wallet.
+    #[arg(long = "wallet-auto-init", default_value_t = false)]
```

Comment:
I think `initialize` is too indirect, it should be explicit that a *new* wallet will be created from a *new* seed.
Perhaps the arg name should change too

Reply:
good point! Renamed to `wallet-auto-create`